fc/jira/backlog_story.py

```diff
@@ -10,6 +10,30 @@ class BacklogStory(BacklogIssue):
     def __init__(self):
         self.duration = None
         self.cost_of_delay = None
+        self.ac = None
+
+    @classmethod
+    def from_json(cls, json: dict, auth: Auth):
+        new_task = super(BacklogStory, cls).from_json(json, auth)
+
+        new_task.ac = json['fields']['customfield_14219']
```

Comment:
Since `customfield_14219` is used twice in this class, can we make it a top level "private" class field?
Meaning between `class BacklogStory(BacklogIssue):` and `def __init__(self):`, you can put a `_acceptance_criteria_jira_field = 'customfield_14219'`.